parser.go

```diff
@@ -6,17 +6,16 @@ import (
 	"time"
 
 	r3 "github.com/golang/geo/r3"
-	dp "github.com/markus-wa/godispatch"
-
 	bit "github.com/markus-wa/demoinfocs-golang/bitread"
 	common "github.com/markus-wa/demoinfocs-golang/common"
 	msg "github.com/markus-wa/demoinfocs-golang/msg"
 	st "github.com/markus-wa/demoinfocs-golang/sendtables"
+	dp "github.com/markus-wa/godispatch"
```

Comment:
Move this up again. The idea is to have all packages of this repo in one group 🙂